pyunlocbox/functions.py

```diff
@@ -867,6 +858,57 @@ def _prox(self, x, T):
         return np.clip(x, 0, np.inf)
 
 
+class proj_spsd(proj):
+    r"""
+    Projection on the Symetric positive semi definite set of matrices
+
+    This function is the indicator function :math:`i_S(z)` of the set S which
+    is zero if `z` is in the set and infinite otherwise. The set S is defined
+    by :math:`\left\{z \in \mathbb{R}^N \mid z \leq 0 \right\}`.
```

Comment:
This paragraph is out-of-date right?

Reply:
Well it is the same as in the other projection function. I agree, it is not super useful and it is repeating.

Reply:
I will suggest something...

Reply:
Well it's useful if the set S is defined as the one the projection operator projects into. ;)

Reply:
Right... Sorry. I missed that.

Reply:
Fixed

Reply:
No problem that's why we review each other. ;)